### PR TITLE
cleanup notes to check before workshop

### PR DESCRIPTION
**Pull request by will-moore**
@pwalczysko These are some notes I made for the OME meeting workshop: Things to check before running the workshop.
Probably useful to have them saved.

### CODE REVIEW

File: maintenance/preparation/cleanup.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ - idr0021 Project has Bulk annotations Table
+ - OMERO.mapr config set for Gene and Key-Value
+ - Figure 2 Aurora-B figure created
+ - Only 1 Project idr0021
+
+
+Cleanup
+-------
+
+ - Delete manually-added client map annotations on images
```
Comment (will-moore) [inline maintenance/preparation/cleanup.md]: Only if added during the workshop demo!

Comment (jburel) [reply maintenance/preparation/cleanup.md]: Could you add that point to the doc?

**Comment by will-moore** [comment]
@pwalczysko I updated this a bit since Friday's run-through.
Anything else we need to add?

File: maintenance/preparation/cleanup.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ - Delete summpary ROI Map annotations from 1st Dataset
+ - Delete Plots attachments on Project
+
+
+Metadata Handling
+=================
+
+ - idr0021 Project has Bulk annotations Table
+ - OMERO.mapr config set for Gene and Key-Value
+ - Figure 2 Aurora-B figure created
```
Comment (pwalczysko) [inline maintenance/preparation/cleanup.md]: We agreed that this Figure will belong to ``trainer-2`` to be able to show the cooperation features on Figure. I have already executed the ``chown`` successfully

Comment (pwalczysko) [reply maintenance/preparation/cleanup.md]: Maybe does not need to add here, not sure.

**Comment by pwalczysko** [comment]
Remove all models (from trainer-1, but I removed all the models). Also removed all the annotations on the svs we intend to use for Orbit

**Comment by will-moore** [comment]
@pwalczysko  I think I've got those covered with 
```
 - Delete any Orbit model annotation(s) (not linked to Images)
 - Delete Orbit ROI annotation on Image
 - Delete ROIs on Image
```

**Comment by pwalczysko** [comment]
@will-moore Opened https://github.com/will-moore/training-scripts/pull/1

**Pull request merged**